docs/user-quick-start.md

```diff
@@ -0,0 +1,299 @@
+enough resources:
+
+- Bare metal or virtual machine with nested virtualization enabled
+- 8 CPUs or more
+- 20 GiB of free memory
+- 100 GiB of free space
+- Internet connection
+- Linux - tested on *Fedora* 37 and 38
+- non-root user with sudo privileges (all instructions are for non-root user)
+
+## Setting up your machine
+
+1. Fork the *Ramen* repository at https://github.com/RamenDR/ramen
+
+1. Clone the source locally
+
+   ```
+   git clone https://github.com/<your_github_id>/ramen.git
+   cd ramen
+   ```
+
+1. Create python virtual environment
+
+   The *Ramen* project use python tool to create and provision test
+   environment and run tests. The create a virtual environment including
+   the tools run:
+
+   ```
+   make venv
+   ```
+
+   This create a virtual environment in `~/.venv/ramen` and a symbolic
+   link `venv` for activating the environment. To activate the
+   environment use:
+   
+   ```
+   source venv
+   ```
+   
+   To exit virtual environment type *deactivate*.
+
+1. Add yourself to the libvirt group (required for minikube kvm2 driver).
+
+   ```
+   sudo usermod -a -G libvirt $(whoami)
+   ```
+
+   Logout and login again for the change above to be in effect.
+
+1. Install minikube - on Fedora you can use::
+
+   ```
+   sudo dnf install https://storage.googleapis.com/minikube/releases/latest/minikube-latest.x86_64.rpm
+   ```
+   
+   Tested with version v1.31.
+   
+1. Install `@virtualization` group - on Fedora you can use:
+
+   ```
+   sudo dnf install @virtualization
+   ```      
+   
+1. Install the `kubectl` tool. See
+   [Install and Set Up kubectl on Linux](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) for details.
+   Tested with version v1.27.4.
+
+1. Install `clusteradm` tool. See
+   [Install clusteradm CLI tool](https://open-cluster-management.io/getting-started/installation/start-the-control-plane/#install-clusteradm-cli-tool) for the details.
+   Tested with v0.6.0.
+
+1. Install `subctl` tool, See
+   [Submariner subctl installation](https://submariner.io/operations/deployment/subctl/) for the details.
+   Tested with version v0.15.2.
+
+1. Install the `velero` tool. See
+   [Velero Basic Install](https://velero.io/docs/v1.11/basic-install/) for the details.
+   Tested with version v1.11.0.
+
+1. Install `helm` tool - on Fedora you can use:
+
+   ```
+   sudo dnf install helm
+   ```
+   
+   Tested with version v3.11.
+
+1. Install the `virtctl` tool. See
+   [virtctl install](https://kubevirt.io/quickstart_minikube/#virtctl)
+   for the details.
+   Tested with version v1.0.1.
+
+1. Install `docker` - on Fedora you can use:
+
+   ```
+   sudo dnf install docker
+   ```
+
+   Add yourself to the `docker` group to allow running docker as root:
+
+   ```
+   sudo usermod -aG docker $USER && newgrp docker
+   ```
+
+   Restart docker service to fix the permissions on the docker daemon
+   socket:
+
+   ```
+   sudo systemctl restart docker
+   ```
```

Comment:
We can remove docker, not needed for testing dr. It is only used for running the drenv tests because of issues in github with podman.